src/Sylius/Bundle/ApiBundle/Resources/config/routing/customer.yml

```diff
@@ -1,8 +1,43 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
+sylius_api_customer_index:
+    path: /
+    methods: [GET]
+    defaults:
+        _controller: sylius.controller.customer:indexAction
+        _sylius:
+            serialization_groups: [Default]
```

Comment:
But I believe the `ResourceController` uses `Default` as serialization group when no group defined, doesn't it?